mysql-test/t/dd_upgrade_partition.test

```diff
@@ -46,6 +46,7 @@ call mtr.add_suppression("Error in checking mysql.slave_master_info repository i
 call mtr.add_suppression("Error creating master info: Error checking repositories.");
 call mtr.add_suppression("Slave: Failed to initialize the master info structure for channel");
 call mtr.add_suppression("Failed to create or recover replication info repositories.");
+call mtr.add_suppression("Cannot load from mysql.* The table is probably corrupted!");
```

Comment:
Why do we need this?

Reply:
This is because 8.0 now has different schema from 5.7. This is normal. mysql_upgrade fixes this.